content/200-concepts/150-data-platform/08-billing/02-plans-and-quotas.mdx

```diff
@@ -0,0 +1,142 @@
+
+</Admonition>
+
+## The first plan of each project
+
+- If a project is created or added in the Prisma Data Platform before the [General Availability](/about/prisma/platform-releases#maturity-levels) of the platform, it starts on the `Early Adopter` plan.
+- If project is created after General Availability, it starts on the `Free` plan.
+
+## Comparison of plans
+
+For plan pricing details, go to our [Pricing](https://cloud.prisma.io/pricing) page.
+
+|                                               | `Early Adopter`                                                  | `Free`                                                   | `Scale`                                                                                                                                                                   | `Enterprise`                                 |
+| --------------------------------------------- | ---------------------------------------------------------------- | -------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------- |
+| Team members                                  | 10                                                               | 3                                                        | Unlimited<br /><br /> <strong>Quota</strong><br /><br />5 <br /><br /><strong>Overage</strong><br /><br />Each additional member at a fixed price                         | Custom                                       |
+| Member roles                                  | • Admin<br />• Viewer<br />• Developer<br />• Collaborator       | Admin only                                               | • Admin<br />• Viewer<br />• Developer<br />• Collaborator                                                                                                                | • Admin<br />• Viewer<br />• Developer<br /> |
+| Environments                                  | 7                                                                | 2                                                        | Unlimited<br /><br /> <strong>Quota</strong><br /><br />5 <br /><br /><strong>Overage</strong><br /><br />Each additional member at a fixed price                         | Custom                                       |
+| **Data Proxy**<br /><br /> - CPU usage        | 100,000,000<br /><br />CPU&nbsp;ms<br /><br />(hard limit)       | 3,000,000<br /><br />CPU&nbsp;ms<br /><br />(hard limit) | Unlimited<br /><br /><strong>Quota</strong><br /><br />30,000,000 CPU&nbsp;ms<br /><br /><strong>Overage</strong><br /><br />Each additional CPU&nbsp;ms at a fixed price | Unlimited<br /><br />Custom pricing          |
```

Comment:
In the EA plan, the proxy also supports only limited concurrency, so it deserves to be called out explicitly. Both Free and Scale do not have this low concurrency limitation.

Reply:
Ah, nevermind. I saw this featured below.